README.md

```diff
@@ -293,6 +293,33 @@ Validators 513571, 513572, 513861 exits successfully initiated
 - `--verbose` - Enable debug mode. Default is false.
 
 
+### Recover your vault data directory and keystores using this command.
+
+```bash
+./operator recover
+```
+
+```
+Enter the mnemonic for generating the validator keys: [Your Mnemonic Here]
+Enter your vault address: 0x3320ad928c20187602a2b2c04eeaa813fa899468
+Enter comma separated list of API endpoints for execution nodes: https://example.com
+Enter comma separated list of API endpoints for consensus nodes: https://example.com
+Enter the network name: goerli
+Vault data and keystores have been successfully recovered.
```

Comment:
Please provide better feedback to the user, like where vault data and keystores are stored.